opencti-platform/opencti-graphql/src/utils/format.js

```diff
@@ -95,7 +95,7 @@ export const observableValue = (stixCyberObservable) => {
     case ENTITY_PROCESS:
       return stixCyberObservable.pid || stixCyberObservable.command_line || 'Unknown';
     case ENTITY_SOFTWARE:
-      return stixCyberObservable.name || 'Unknown';
+      return stixCyberObservable.name || stixCyberObservable.cpe || stixCyberObservable.swid || 'Unknown';
```

Comment:
We could create a function returning the truthy submitted argument or the second argument as the default value.

Something like : 
```
function handleValue(attributes: Array<Attribute>, defaultValue: string): Attribute | string {
  return attributes.find(attribute => !!attribute) || defaultValue
}
```
And use it for every switch case.

Further, refactoring could be the use of this function in Map with switch cases as keys.

Reply:
thanks for the suggestion, the refacto will be made later when we have found a way to no longer have the list duplicated in the front and back.